runtime/libs/circle-schema/include/circle_schema_generated.h

```diff
@@ -1789,9 +1796,9 @@ inline const char *const *EnumNamesBuiltinOperator()
 
 inline const char *EnumNameBuiltinOperator(BuiltinOperator e)
 {
-  if (::flatbuffers::IsOutRange(e, BuiltinOperator_ROPE, BuiltinOperator_REDUCE_WINDOW))
+  if (::flatbuffers::IsOutRange(e, BuiltinOperator_RUN_MODEL, BuiltinOperator_REDUCE_WINDOW))
```

Comment:
```suggestion
  if (::flatbuffers::IsOutRange(e, BuiltinOperator_MIN, BuiltinOperator_MAX))
```

How about this to maintain the code easily?

Reply:
This file is auto generated code by flatc.